app/assets/stylesheets/application.scss

```diff
@@ -312,10 +326,17 @@ main {
           display: inline-block;
         }
 
-        a {
+        button {
           @include core-16;
           padding: 0  5px 0 5px;
           display: inline-block;
+          -webkit-font-smoothing: antialiased;
+          -moz-osx-font-smoothing: grayscale;
+          font-size: 100%;
+          border-width: 0;
+          color: $govuk-blue;
+          background: none;
```

Comment:
Fair bit of repetition between this block and the [`button` block on line 281](https://github.com/alphagov/manuals-frontend/pull/642/files#diff-f859e2848b07324f808ce1a5ccd9fcbaR281) - is there any way those can be combined?

Reply:
thanks, will add shared styles in a placeholder